ArraysHomework/ArraysHomework.cs

```diff
@@ -55,24 +65,39 @@ public void Arrays()
 
             // Swap the last and first value of this array.
             string[] swapMyValues = { "first", "middle", "last" };
+            swapMyValues[0] = swapMyValues[2];
+            swapMyValues[2] = swapMyValues[0];
             
+
             Assert.IsTrue(swapMyValues[0] == "last");
             Assert.IsTrue(swapMyValues[2] == "first");
 
             //Combine these two arrays into a 3rd array (put your code after the for loop);
             int[] array1 = new int[25];
             int[] array2 = new int[25];
-
+            int[] combined = new int [50];
             // just a little fun to fill them
             int up = 0;
             int down = 24;
-            for (; up < array1.Length && down >= 0; up++, down--)
+            int combined = 50;
+            for (int i = 0; up < array1.Length && down >= 0; up++, down--)
             {
                 array1[up] = up;
                 array2[down] = down;
-                
             }
 
+            for (up < array1.Length && down >= combined.Length; up++, down--)
+            {
+                combined[up] = array1[up];
+                combined[down] = array2[down];
+            }
+
+            int j
```

Comment:
This statement isn't terminated (no semicolon).
And in order to use `j++` on line 99, you need to assign an initial value to j.